tests/pfcwd/test_pfcwd_warm_reboot.py

```diff
@@ -150,7 +150,8 @@ def storm_defer_setup(self):
         """
         self.pfc_wd['storm_start_defer'] = random.randrange(120)
         self.pfc_wd['storm_stop_defer'] = random.randrange(self.pfc_wd['storm_start_defer'] + 5, 125)
-        self.max_wait = max(self.max_wait, self.pfc_wd['storm_stop_defer'])
+        # Added 20 sec to max_wait as sometimes it's not enough and test fail due to runtime error
+        self.max_wait = max(self.max_wait, self.pfc_wd['storm_stop_defer']) + 20
```

Comment:
20s is too long. can you reduce it?

Reply:
@neethajohn Inside `join_all()` function each 0.1 sec is checked whether thread is alive, and its quite possible that only 5 seconds will be used out of 20 seconds that I added. As soon as all threads will be stopped, test case will continue running. Anyway, if its too much I could reduce it to 10 -15 seconds ?

Reply:
10sec should be good

Reply:
@neethajohn Done